server.js

```diff
@@ -20,9 +30,42 @@ app.use(bodyParser.json())
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Welcome to Happy Thoughts API by Jihyun')
+})
+
+app.get('/GET', async (req, res) => {
+  const happyThoughts = await Thought.find().sort({ createdAt: 'desc' }).limit(20).exec()
+  if (happyThoughts) {
+    res.status(201).json(happyThoughts)
+  } else {
+    res.status(401).json({ message: 'Could not find Happy Thoughts' })
+  }
+
 })
 
+app.post('/POST', async (req, res) => {
```

Comment:
You misunderstood the requirements( post /). this line should be: app.post('/', async (req, res) => {